src/customizations/volto/components/theme/View/FileView.jsx

```diff
@@ -35,7 +35,7 @@ const FileView = ({ content }) => (
         <Col className="card-wrapper card-teaser-wrapper">
           <div className="genericcard card card-teaser shadow p-4 mt-3 rounded">
             <div className="card-body">
-              <h5 className="card-title">{content.file.filename}</h5>
+              <h2 className="card-title h5">{content.file.filename}</h2>
```

Comment:
http://localhost:3000/novita/comunicati/giornata-nazionale-per-lintegrita-in-sanita-levento-il-6-aprile-a-roma-sara-visibile-anche-on-line/allegati/programma_giornata_nazionale_integrita_sanita_6_aprile_2022.pdf#:~:text=Programma_Giornata_Nazionale_Integrita_Sanita_6_aprile_2022.pdf-,Programma_Giornata_Nazionale_Integrita_Sanita_6_aprile_2022,-.pdf